packages/ketcher-react/src/script/editor/tool/select.ts

```diff
@@ -16,28 +16,21 @@
 
 import {
   Action,
-  SGroup,
   fromAtomsAttrs,
   fromBondsAttrs,
-  fromItemsFuse,
-  fromMultipleMove,
   fromTextDeletion,
   fromTextUpdating,
-  getHoverToFuse,
-  getItemsToFuse,
-  FunctionalGroup,
-  fromSimpleObjectResizing,
-  fromArrowResizing,
-  ReStruct,
-  ReSGroup
+  FunctionalGroup
 } from 'ketcher-core'
 
 import LassoHelper from './helper/lasso'
-import { atomLongtapEvent } from './atom'
 import SGroupTool from './sgroup'
-import utils from '../shared/utils'
 import { xor } from 'lodash/fp'
 import { Editor } from '../Editor'
+import { startSelecting } from '../utils/selectItems/startSelecting'
+import { finishSelecting } from '../utils/selectItems/finishSelecting'
+import { continueSelecting } from '../utils/selectItems/continueSelecting'
+import { cancelSelecting } from '../utils/selectItems/cancelSelecting'
```

Comment:
onMouse controllers used both in select.ts and sgroup.ts moved to dedicated utils